examples.md

```diff
@@ -385,7 +385,13 @@ Replace `~/.local/share/renv` with the correct `path` if not using Ubuntu.
     restore-keys: |
       ${{ runner.os }}-gems-
 ```
-When dependencies are installed later in the workflow, we must specify the same path for the bundler.
+When dependencies are installed later in the workflow, we must use the `--deployment` flag to install gems to `vendor/bundle`.
+
+```yaml
+- name: Bundle install
+  run: bundle install --deployment --jobs 4 --retry 3
+```
+If you don’t want to run Bundler in deployment mode, you can instead use `bundle config`.
```

Comment:
Could we make it clear that you don't need both of these Bundle install steps?

Reply:
What do you think about either completely dropping this sentence or replacing it with the suggestion below?

> Note: Bundler’s [deployment mode](https://bundler.io/v2.0/man/bundle-install.1.html#DEPLOYMENT-MODE) requires an up-to-date `Gemfile.lock`.

I think having a `Gemfile.lock` is a good practice and the majority of projects do have one. In my opinion, we don’t necessarily need to complicate this example by directly providing an alternative. (As for out-of-date lockfiles, `--deployment` is arguably a helpful safeguard.)